tests/run-make/x86_64-fortanix-unknown-sgx-lvi/script.sh

```diff
@@ -48,7 +49,8 @@ build
 
 check "unw_getcontext" unw_getcontext.checks
 check "__libunwind_Registers_x86_64_jumpto" jumpto.checks
-check 'std::io::stdio::_print::[[:alnum:]]+' print.checks
+check 'std::io::stdio::_print::[[:alnum:]]+' print.with_frame_pointes.checks ||
+    check 'std::io::stdio::_print::[[:alnum:]]+' print.without_frame_pointes.checks
```

Comment:
```suggestion
check 'std::io::stdio::_print::[[:alnum:]]+' print.with_frame_pointers.checks ||
    check 'std::io::stdio::_print::[[:alnum:]]+' print.without_frame_pointers.checks
```

Reply:
Oh good catch! I was under the impression that the script would fail if if referenced a file that didn't exist. There's the `set -e` option at line 2, but that doesn't work well with `sh`. I already filed #121685 last week to clean up this script. It also changes the shell to `bash`. I've only updated the filenames here.